bin/install.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/sh
+set -e
+
+# Adapted/Copied from https://github.com/daveshanley/vacuum/blob/main/bin/install.sh
+
+# Default to /usr/local/bin, but fall back to $HOME/.local/bin if /usr/local/bin is not writable
+if [ -w "/usr/local/bin" ]; then
```

Comment:
I'm not a huge fan of installing by piping from `curl`'d scripts in the first place, but I think if it were me, I would default to `$HOME/.local/bin` first, since it requires less privs and it's inherently more secure. Just my opinion.

Reply:
Done